Source/Scene/Batched3DModel3DTileContent.js

```diff
@@ -0,0 +1,455 @@
+/*global define*/
+define([
+        '../Core/Color',
+        '../Core/defaultValue',
+        '../Core/defined',
+        '../Core/defineProperties',
+        '../Core/deprecationWarning',
+        '../Core/destroyObject',
+        '../Core/DeveloperError',
+        '../Core/getAbsoluteUri',
+        '../Core/getBaseUri',
+        '../Core/getStringFromTypedArray',
+        '../Core/RequestType',
+        './Cesium3DTileBatchTable',
+        './Cesium3DTileFeature',
+        './Cesium3DTileFeatureTable',
+        './getAttributeOrUniformBySemantic',
+        './Model'
+    ], function(
+        Color,
+        defaultValue,
+        defined,
+        defineProperties,
+        deprecationWarning,
+        destroyObject,
+        DeveloperError,
+        getAbsoluteUri,
+        getBaseUri,
+        getStringFromTypedArray,
+        RequestType,
+        Cesium3DTileBatchTable,
+        Cesium3DTileFeature,
+        Cesium3DTileFeatureTable,
+        getAttributeOrUniformBySemantic,
+        Model) {
+    'use strict';
+
+    /**
+     * Represents the contents of a
```

Comment:
This should have a reference back to `Cesium3DTileContent` and explicitly mention it implements that interface in the summary.